akka-http-core/src/main/resources/reference.conf

```diff
@@ -101,6 +101,10 @@ akka.http {
     # holding the clients (remote) IP address.
     remote-address-header = off
 
+    # Enables/disables the addition of a remote-address attribute in HttpRequest
+    # holding the clients (remote) IP address.
```

Comment:
```suggestion
    # holding the clients (remote) IP address. This is preferred over `remove-address-header`
    # because it cannot be confused with a real header.
```